src/main/fc/fc_msp.c

```diff
@@ -153,6 +156,9 @@ static const box_t boxes[CHECKBOX_ITEM_COUNT] = {
     { BOX3DDISABLESWITCH, "DISABLE 3D SWITCH", 29},
     { BOXFPVANGLEMIX, "FPV ANGLE MIX", 30},
     { BOXBLACKBOXERASE, "BLACKBOX ERASE (>30s)", 31 },
+    { BOXCAMERAWIFI, "CAMERA WIFI", 32},
+    { BOXCAMERAPOWER, "CAMERA POWER", 33},
+    { BOXCAMERACHANGEMODE, "CAMERA CHANGE MODE", 34 },
```

Comment:
I'm not sure we should have such specific camera functions named like this.

I would prefer something more generic so that we don't start adding loads of really specific code.

I think something like this should be OK for now and for the future.
```
+    { BOXCAMERA1, "CAMERA CONTROL 1", 32},
+    { BOXCAMERA2, "CAMERA CONTROL 2", 33},
+    { BOXCAMERA3, "CAMERA CONTROL 3", 34 },
```

The configurator code can show appropriate labels/documentation when 'RC Split' is chosen by the user if need be.